tests/swe_time_equ.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Basic test
+--SKIPIF--
+<?php
+if (!extension_loaded('swephp')) {
+    echo 'skip';
+}
+?>
+--FILE--
+<?php
+swe_set_ephe_path('./sweph/ephe');
+$delta = swe_time_equ(2452275.5);
+$delta['te'] = round($delta['te'], 6);
```

Comment:
If more tests are added in the future which require special treatment like this, it may be valuable to create some functions or a class that can be consumed by the tests to expedite test writing. The remaining ones in this PR don't seem to justify that extra work at this time, however.